packages/webview_flutter/webview_flutter_android/example/android/app/src/androidTest/java/io/flutter/plugins/webviewflutterexample/BackgroundColorTest.java

```diff
@@ -0,0 +1,59 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.webviewflutterexample;
+
+import static androidx.test.espresso.flutter.EspressoFlutter.onFlutterWidget;
+import static androidx.test.espresso.flutter.action.FlutterActions.click;
+import static androidx.test.espresso.flutter.matcher.FlutterMatchers.withText;
+import static androidx.test.espresso.flutter.matcher.FlutterMatchers.withValueKey;
+import static org.junit.Assert.assertEquals;
+
+import android.graphics.Bitmap;
+import android.graphics.Color;
+import androidx.test.core.app.ActivityScenario;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.runner.screenshot.ScreenCapture;
+import androidx.test.runner.screenshot.Screenshot;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(AndroidJUnit4.class)
+public class BackgroundColorTest {
+  @Rule
+  public ActivityTestRule<DriverExtensionActivity> myActivityTestRule =
+      new ActivityTestRule<>(DriverExtensionActivity.class, true, false);
+
+  @Before
+  public void setUp() {
+    ActivityScenario.launch(DriverExtensionActivity.class);
+  }
+
+  @Test
+  public void backgroundColor() {
+    onFlutterWidget(withValueKey("ShowPopupMenu")).perform(click());
```

Comment:
This is failing in CI:
```
androidx.test.espresso.AppNotIdleException: Looped for 5404 iterations over 60 SECONDS. The following Idle Conditions failed .
at dalvik.system.VMStack.getThreadStackTrace(Native Method)
at java.lang.Thread.getStackTrace(Thread.java:1736)
at androidx.test.espresso.base.DefaultFailureHandler.getUserFriendlyError(DefaultFailureHandler.java:96)
at androidx.test.espresso.base.DefaultFailureHandler.handle(DefaultFailureHandler.java:59)
at androidx.test.espresso.ViewInteraction.waitForAndHandleInteractionResults(ViewInteraction.java:322)
at androidx.test.espresso.ViewInteraction.desugaredPerform(ViewInteraction.java:178)
at androidx.test.espresso.ViewInteraction.perform(ViewInteraction.java:119)
at androidx.test.espresso.flutter.EspressoFlutter$WidgetInteraction.performInternal(EspressoFlutter.java:142)
at androidx.test.espresso.flutter.EspressoFlutter$WidgetInteraction.perform(EspressoFlutter.java:107)
at io.flutter.plugins.webviewflutterexample.BackgroundColorTest.backgroundColor(BackgroundColorTest.java:38)
at java.lang.reflect.Method.invoke(Native Method)
```

Does it work locally? (I haven't had a chance to test it locally myself yet.)

Reply:
Answering my own question since I had a few minutes at my computer: it does indeed. So the issue is specific to the FTL setup.

@blasten @bparrishMines Any idea why the very first attempt to interact with a widget would work locally (via `native-test`) but time out when run via FTL?

(If we don't find a straightforward solution, we can temporarily disable this test and file an issue to re-enable it; we want this running in CI ideally, but we don't need to block the feature on FTL debugging.)

Reply:
Filed https://github.com/flutter/flutter/issues/94748 for follow-up.